diff --git a/scripts/glue/silver_customers.py b/scripts/glue/silver_customers.py
index 6fc0b95..25654cc 100644
--- a/scripts/glue/silver_customers.py
+++ b/scripts/glue/silver_customers.py
@@ -1,4 +1,5 @@
 import sys
+import os
 from awsglue.context import GlueContext
 from awsglue.job import Job
 from awsglue.utils import getResolvedOptions
@@ -24,7 +25,7 @@
 job = Job(glue_context)
 job.init(args["JOB_NAME"], args)
 
-bucket = args.get("S3_BUCKET", "default-dms-bucket")
+bucket = os.environ.get('S3_BUCKET', 'analytics-curated')
 raw_prefix = args["RAW_PREFIX"].rstrip("/")
 silver_prefix = args["SILVER_PREFIX"].rstrip("/")
